bin/public/installGlassfish4.py

```diff
@@ -20,52 +20,33 @@
 __status__ = "Production"
 
 import os
-import re
-import shutil
-import stat
-import sys
-import traceback
-import time
-
 import app
-import config
 import general
-import version
-import iptables
-import install
 from general import x
-from scopen import scOpen
-import socket
 
 # The version of this module, used to prevent the same script version to be
 # executed more then once on the same host.
 SCRIPT_VERSION = 2
 
-# NOTE: Remember to change path in "var/glassfish/glassfish-3.1.1"
+
+# NOTE: Remember to change path in "var/glassfish/glassfish-4.1"
 GLASSFISH_VERSION      = "glassfish-4.1"
 GLASSFISH_INSTALL_FILE = GLASSFISH_VERSION + ".zip"
 GLASSFISH_REPO_URL     = "http://packages.fareoffice.com/glassfish/" + GLASSFISH_INSTALL_FILE
-GLASSFISH_INSTALL_PATH = "/usr/local/glassfish4"
-GLASSFISH_DOMAINS_PATH = GLASSFISH_INSTALL_PATH + "/glassfish/domains/"
 GLASSFISH_USER         = "glassfish"
 
-#Icinga plugins directory
+# Icinga plugins directory
 ICINGA_PLUGINS_DIR = "/usr/lib64/nagios/plugins/"
 
-# The directory where JAVA stores temporary files.
-# Default is /tmp, but the partion that dir is stored on is set to "noexec", and
-# java requires to exeute code from the java tmp dir.
-JAVA_TEMP_PATH = GLASSFISH_INSTALL_PATH + "tmp"
-
 # http://www.oracle.com/technetwork/java/javase/downloads/index.html
-JDK_INSTALL_FILE = "jdk-8u40-linux-x64.tar.gz"
+JDK_INSTALL_FILE = "jdk-8u66-linux-x64.tar.gz"
```

Comment:
I don't think it makes sense to maintain a seperate java installation for glassfish. I think it syco install-java-rpm should be run before this installation instead.